sky/jobs/state.py

```diff
@@ -586,12 +633,15 @@ def set_cancelling(job_id: int, callback_func: CallbackType):
         rows = cursor.execute(
             """\
             UPDATE spot SET
-            status=(?), end_at=(?)
```

Comment:
Why do we remove this? Is it because we want to only set the `end_at` for the terminal state?

Reply:
Yes.